### ISSUE TITLE
mbelib calls exit() in mbe_synthesizeSpeechf()

### ISSUE THREAD

**Issue by pbarfuss** [szechyjs/mbelib#13]
This is absolutely awful, especially since it calls it for an invalid parameter. There's no way to do anything else from within a program linking against it, because, well, exit() was just called.

The best solution if you want to modify the API is to make mbe_synthesizeSpeechf() return an int / error code, with it returning, say, -1 in that case and, say, 0 otherwise.

The best solution if you don't want to modify the API, and also personally the one I recommend, is simply if the value is invalid, print a warning and then just set it to 3 - this gives you 4 unvoiced bands, which is comparable to the stupid FFT-of-white-noise method DVSI wants you to do, and is also the DSD default.

In other words,
  if ((uvquality < 1) || (uvquality > 64))
    {
      printf ("\nmbelib:  Error - uvquality must be within the range 1 - 64\n");
      exit (1);
    }

Becomes:
  if ((uvquality < 1) || (uvquality > 64))
    {
      printf ("\nmbelib:  Error - uvquality must be within the range 1 - 64, setting to default value of 3\n");
      uvquality = 3;
    }

**Comment by szechyjs** [comment]
Would you like to submit a pull request and claim credit? Else, I can make the change you proposed.

**Comment by pbarfuss** [comment]
Sure. Still kind of getting used to git and github, just submitted one.
